mapper/pojo-base/src/main/java/org/hibernate/search/mapper/pojo/mapping/definition/annotation/processing/impl/MarkerBindingProcessor.java

```diff
@@ -28,8 +29,16 @@
 	@Override
 	public void process(PropertyMappingStep mapping, MarkerBinding annotation,
 			PropertyMappingAnnotationProcessorContext context) {
-		MarkerBinder binder = createBinder( annotation.binder(), context );
-		mapping.marker( binder );
+		MarkerBinderRef markerBinderRef = annotation.binder();
+		MarkerBinder binder = createBinder( markerBinderRef, context );
+
+		if ( markerBinderRef.params() != null ) {
+			Map<String, Object> params = ParameterAnnotationsReader.toMap( markerBinderRef.params() );
```

Comment:
What's the point of exposing a method in the `context` if you don't use it? :)

```suggestion
			Map<String, Object> params = context.toMap( markerBinderRef.params() );
```